docs/community/developer_guide.mdx

```diff
@@ -121,12 +139,13 @@ This section needs a lot of love and automation 🙂
         gpg --verify $file
     done
     ```
-1. Click the "Edit release" button on the release, and add the `.asc` files created in the previous step. Edit the release description for change logs and also add a note indicating your GPG key used to sign the artifacts. Example (replace with your fingerprint etc.):
+1. Click the "Edit release" button on the release, and add the `.asc` files created in the previous step. Edit the release description to include change logs and a note indicating releaser's GPG key used to sign the artifacts. Example (replace with releaser's fingerprint etc.):
```

Comment:
```suggestion
1. Click the "Edit release" button on the release, and add the `.asc` files created in the previous step. Edit the release description to include change logs and a note indicating the releaser's GPG key used to sign the artifacts. See an example:
```

Reply:
We may not need to indicate to replace it since this is an example and it's intuitive enough